tools/WinObjC.Language/WinObjC.Language.Packaging.targets

```diff
@@ -3,11 +3,12 @@
 
   <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
       <ItemGroup>
-          <!-- Inlcude *all* of /include/  and /include_next/ folder.  -->
-          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*"/>
+          <!-- Include *all* of /include/, /include/xplat/ and /include_next/ folder.  -->
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*" Exclude="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
```

Comment:
Interesting apparently putting asteriks around words italicizes or bolds them.

---
In reply to: [111030228](https://github.com/Microsoft/WinObjC/pull/2493#discussion_r111030228) [](ancestors = 111030228)